### PR TITLE
Closes #362: Add some documentation about which version of az-icons to use

### LINKED ISSUE #362: Add some documentation about which version of az-icons to use under different circumstances.
## User Story(s)

As a developer or site maintainer, i'd like to be certain that my version of az-icons is compatible with my site. Also, I'd like to know when I should update the az-icons version.
…

## Conditions of satisfaction
- [ ] Should explain a bit about what the versions mean and how to use them with example contexts.

Generally, the idea is to use the main branch version for development work, and switch to a pinned version when development is done.

Latest is good for when you need to use icons that have been released, and you are following the az-icons project and know what has changed and how those changes will affect your site.

Pinned versions are good for any project that wants to be sure to not update az-icons if you don't purposefully change it.

### PR DESCRIPTION
**Pull request by geijutsuka**
Pinned and Latest releases are not available yet, but the wording is there to inform which to use once they are available.

### CODE REVIEW

**Comment by geijutsuka** [comment]
Potentially blocked by: https://github.com/az-digital/az-icons/issues/6 
If the AZ Icons issue is resolved before merging this, I will need to take out the line which states:
`(Latest vs. Pinned releases coming soon)`

**Review by mmunro-ltrr — APPROVED** [review]
Re-tested on a local build.

**Review by camikazegreen — APPROVED** [review]

**Review by trackleft — APPROVED** [review]

**Pull request merged**
